data_steward/tools/combine_ehr_rdr.py

```diff
@@ -46,10 +46,30 @@
 """
 import logging
 
+from jinja2 import Environment
+
+import app_identity
 import bq_utils
 import common
-from constants.tools import combine_ehr_rdr as combine_consts
 import resources
+from constants.tools import combine_ehr_rdr as combine_consts
+from utils import bq as bq
+
+jinja_env = Environment(
+    # help protect against cross-site scripting vulnerabilities
+    autoescape=True,
+    # block tags on their own lines
+    # will not cause extra white space
+    trim_blocks=True,
+    lstrip_blocks=True,
+    # syntax highlighting should be better
+    # with these comment delimiters
+    comment_start_string='--',
+    comment_end_string=' --')
+
+COPY_PID_RID_QUERY = jinja_env.from_string("""
+select * from `{{project}}.{{dataset}}.{{pid_rid_table}}`
+""")
```

Comment:
I'm hesitant about creating the `_deid_map` table in the combined dataset code.  The combined dataset has no reason to know about a mapping table created for and used later in the pipeline (this is a criticism of the mapping table's historical placement).  It would be better (not part of your PR) to store this mapping table in a sandbox dataset in the future.  So, I'm concerned that putting this table copy code in this module would make that transition harder in the future.  It would make more sense to either create this mapping table from `curation/data_steward/tools/run_deid.py` where it can run once, or it can replace the `create_participant_mapping_table()` function in `curation/data_steward/deid/aou.py`.  This has the disadvantage of running once for every table that is deidentified.